_includes/footer.html

```diff
@@ -1,19 +1,145 @@
-<footer class="footer bg-navy">
-  {% include i18n_dropdown_mobile.html %}
-  <div class="container cntnr-wide py1 px2 lg-px0{% if site.languages.size > 1 %} sm-py0{% endif %}">
-    <div class="flex flex-center">
-      <div class="flex flex-center">
-        {% include i18n_dropdown_desktop.html %}
-        <a href="{{ '/help/' | relative_url }}" class="caps h6 white text-decoration-none mr3 {% if page.url == '/help/' %}bold{% endif %}">{% t meta.help.title %}</a>
-        <a href="{{ '/contact/' | relative_url }}" class="caps h6 white text-decoration-none mr3 {% if page.url == '/contact/' %}bold{% endif %}">{% t meta.contact.title %}</a>
-        <a href="{{ '/policy/' | relative_url }}" class="caps h6 white text-decoration-none {% if page.url == '/policy/' %}bold{% endif %}">{% t meta.privacy_and_security.title %}</a>
+<footer class="footer">
+  <div class="footer-nav">
+    <div class="container">
+      <!-- apply to .footer-nav -->
+      <div class="navigation">
+        <nav>
+          <div class="grid">
+            <!-- apply to nav -->
+            <section>
+              <!-- replace with uls -->
+              <h6>{% t nav.groups.agencies %}</h6>
+              <a class="h6" target="_blank" href="https://partners.login.gov"
+                >{% t nav.become_a_partner %}</a
+              >
+              <a class="h6" target="_blank" href="https://developers.login.gov"
+                >{% t nav.developer_guide %}</a
+              >
+            </section>
+            <section>
+              <h6>{% t nav.groups.learn %}</h6>
+              <a class="h6" href="{{ site.baseurl }}/about/"
+                >{% t nav.about_us %}</a
```

Comment:
what's the rule or logic we use for when to break the closing `>` on to a new line? compared to this line below where each attribute is on its own line?

```
              <a
                class="h6"
                target="_blank"
```

Reply:
If three or more attributes, drops to its own line. See line 27.

Reply:
Not sure if it's what @zachmargolis was getting at, but the placement of the `>` appears quite strange, where I'd expect even if there's a distinction of how many attributes before breaking onto own lines, if it's the case that there's few enough attributes to stay on one line, the `>` would be kept here as well:

```html
<a class="h6" href="{{ site.baseurl }}/about/">
  {% t nav.about_us %}
</a>
```

Is this a consequence of Prettier? If so, I also wonder about...

- How good is Prettier at handling ERB templating? ([related](https://github.com/prettier/plugin-ruby/blob/master/CONTRIBUTING.md#do-you-support-erb-files-htmlerb-jserb-etc))
- What's the developer story for ensuring this remains consistent? For example, when I try to run Prettier in VSCode, I'm seeing warnings about missing a plugin. Do I need to install something?